[2025-03-16T04:45:47.224+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T04:45:47.237+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T04:45:39.083469+00:00 [queued]>
[2025-03-16T04:45:47.245+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T04:45:39.083469+00:00 [queued]>
[2025-03-16T04:45:47.245+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T04:45:47.255+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.ingest_raw_data> on 2025-03-16 04:45:39.083469+00:00
[2025-03-16T04:45:47.263+0000] {standard_task_runner.py:72} INFO - Started process 2441 to run task
[2025-03-16T04:45:47.266+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.ingest_raw_data', 'manual__2025-03-16T04:45:39.083469+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpxksb09m3']
[2025-03-16T04:45:47.268+0000] {standard_task_runner.py:105} INFO - Job 192: Subtask bronze_layer_group.ingest_raw_data
[2025-03-16T04:45:47.315+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T04:45:39.083469+00:00 [running]> on host c891f3a5df31
[2025-03-16T04:45:47.407+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.ingest_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T04:45:39.083469+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T04:45:39.083469+00:00'
[2025-03-16T04:45:47.409+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T04:45:47.413+0000] {ingest_raw_data.py:139} INFO - Scanning for files in topics/tracking.user_behavior.validated/year=2025/month=01...
[2025-03-16T04:45:47.414+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T04:45:47.416+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T04:45:47.417+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T04:45:47.417+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T04:45:47.967+0000] {ingest_raw_data.py:150} INFO - Loading checkpoint data...
[2025-03-16T04:45:48.049+0000] {ingest_raw_data.py:154} INFO - No new files to process
[2025-03-16T04:45:48.049+0000] {python.py:240} INFO - Done. Returned value was: {'data': [], 'skipped_files': 1}
[2025-03-16T04:45:48.073+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T04:45:48.073+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.ingest_raw_data, run_id=manual__2025-03-16T04:45:39.083469+00:00, execution_date=20250316T044539, start_date=20250316T044547, end_date=20250316T044548
[2025-03-16T04:45:48.123+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T04:45:48.143+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T04:45:48.146+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
